torchdynamo/convert_frame.py

```diff
@@ -292,6 +292,8 @@ def debug_print(prefix):
                     if attempt > 100:
                         unimplemented("100+ RestartAnalysis() calls")
                 except exc.SkipFrame:
+                    if one_graph and config.debug:
+                        print("ERROR: No graph captured with one_graph=True")
```

Comment:
Another reason we should add a log here :)

Reply:
Yeah I'm going to handle this in the logging PR.